tests/TestCase/ORM/Behavior/EnumBehaviorTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         5.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\ORM\Behavior;
+
+use Cake\TestSuite\TestCase;
+use TestApp\Model\Enum\ArticleStatus;
+
+/**
+ * EnumBehavior test case
+ */
+class EnumBehaviorTest extends TestCase
+{
+    /**
+     * @var \Cake\ORM\Table
+     */
+    protected $articles;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $articles = $this->getTableLocator()->get('Articles');
+        $this->articles = $articles->addBehavior('Enum', [
+            'fieldMap' => [
+                'published' => ArticleStatus::class,
+            ],
+        ]);
+    }
+
+    /**
+     * Check adding entity fields with an enum instance
+     */
+    public function test_add_with_enum(): void
```

Comment:
All test methods need to be pascalCased.

Reply:
I always run into this 🙈